# Aula 01 (2021-06-01)
- Importação da biblioteca pandas e criação do DataFrame

In [6]:
import pandas as pd
df = pd.read_csv('datasets/kc_house_data.csv')

## Há duplicatas de ids?
- Os IDs são valores únicos para cada casa, não sendo possível que existam mais de uma casa com mesmo ID. 
- Para fazer essa verificação, a biblioteca pandas possui um método que facilita esse processo

In [12]:
df['id'].duplicated().value_counts()

False    21436
True       177
Name: id, dtype: int64

- Com isso, verificamos que existem 177 duplicatas e 21436 IDs únicos;
- É possível que as duplicatas existam porque o preço de alguns imóveis foram atualizados e os valores antigos não foram deletados.

In [15]:
df.drop_duplicates(subset='id', keep='last', inplace=True)